ush/wave_prnc_ice.sh

```diff
@@ -133,29 +106,19 @@
 
 # 2.d Run through preprocessor wave_prep
 
-  set +x
-  echo '   Run through preprocessor ...'
-  echo ' '
-  set_trace
+  printf "   Run through preprocessor ...\n"
 
   cp -f ${DATA}/ww3_prnc.ice.$WAVEICE_FID.inp.tmpl ww3_prnc.inp
 
   export pgm="${NET,,}_ww3_prnc.x"
   source prep_step
 
   "${EXECgfs}/${pgm}" 1> prnc_${WAVEICE_FID}_${cycle}.out 2>&1
-  export err=$?; err_chk
-  if [ "$err" != '0' ]
+  export err=$?
+  if [[ ${err} -ne 0 ]]
   then
     cat prnc_${WAVEICE_FID}_${cycle}.out
-    set +x
-    echo ' '
-    echo '******************************************** '
-    echo '*** WARNING: NON-FATAL ERROR IN ww3_prnc *** '
-    echo '******************************************** '
-    echo ' '
-    set_trace
-    echo "WARNING: NON-FATAL ERROR IN ww3_prnc."
+    echo "ERROR: failure in ${pgm}"
```

Comment:
Why not WARNING?

Reply:
Because it is exiting with a non-zero status.  This will cause a FATAL ERROR in the wave init job.